posthog/tasks/usage_report.py

```diff
@@ -650,10 +688,12 @@ def convert_team_usage_rows_to_dict(rows: list[Union[dict, tuple[int, int]]]) ->
         if isinstance(row, dict) and "team_id" in row:
             # Some queries return a dict with team_id and total
             team_id_map[row["team_id"]] = row["total"]
-        else:
+        elif isinstance(row, list | tuple) and len(row) == 2:
             # Others are just a tuple with team_id and total
             team_id_map[int(row[0])] = row[1]
-
+        else:
+            # Handle the case where row is an integer
+            team_id_map[row] = 1  # or some other default value
```

Comment:
```suggestion
            team_id_map[row] = 1
```

Is this the default we want? When would we end up with this default?

Reply:
managed to fix it so this is no longer needed!